mecra.csl

```diff
@@ -0,0 +1,1467 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" delimiter-precedes-et-al="never" delimiter-precedes-last="never" et-al-min="3" et-al-use-first="1" page-range-format="expanded">
+  <info>
+    <title>Mecra</title>
+    <id>http://www.zotero.org/styles/mecra</id>
+    <link href="http://www.zotero.org/styles/mecra" rel="self"/>
+    <link href="http://mecra-sistemi.blogspot.com/" rel="documentation"/>
+    <author>
+      <name>Muhammet Tarakçı</name>
+      <email>muhammettarakci@gmail.com</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="social_science"/>
+    <summary xml:lang="tr">Mecra, Türkiye'de yaygın akademik yazım tercihleri gözetilerek geliştirilmiş bir şablondur. </summary>
+    <updated>2019-08-07T15:51:11+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="collection-editor">ed.</term>
+      <term name="editorial-director">ed.</term>
+    </terms>
+  </locale>
+  <locale xml:lang="tr">
+    <terms>
```

Comment:
I didn't check, but are all these terms actually necessary? Aren't most of these in the Turkish locale?